### PR TITLE
librbd/cache/pwl: Rename config options

### PR DESCRIPTION
**Pull request by MahatiC**
Signed-off-by: Mahati Chamarthy <mahati.chamarthy@intel.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by MahatiC** [comment]
@dillaman please review

**Review by dillaman — APPROVED** [review]
lgtm -- thx!

File: src/librbd/plugin/WriteLogImageCache.cc [hunk 1]
```diff
@@ -42,7 +42,7 @@ void WriteLogImageCache<I>::init(I* image_ctx, Api<I>& api,
                                  PluginHookPoints& hook_points_list,
                                  Context* on_finish) {
   bool rwl_enabled = image_ctx->config.template get_val<bool>(
-    "rbd_rwl_enabled");
+    "rbd_persistent_cache_enabled");
```
Comment (lixiaoy1) [inline src/librbd/plugin/WriteLogImageCache.cc:45]: Do we have such option? I thought it was changed to `rbd_persistent_cache_mode`.

Comment (MahatiC) [reply src/librbd/plugin/WriteLogImageCache.cc:45]: I misunderstood our convo. I thought you needed it. As of today `rbd_persistent_cache_enabled` doesn't exist in `src/common/options.cc`. Can remove this option and this code can be replaced with `is_pwl_enabled` util. Will fix this.

**Comment by MahatiC** [comment]
retest this please

**Pull request merged**